code/messaging/src/main/AndroidManifest.xml

```diff
@@ -25,5 +25,15 @@
                 android:name="com.adobe.marketing.mobile.messaging.MessagingExtension"
                 android:value="Extension" />
         </service>
+
+        <provider
+            android:name="androidx.core.content.FileProvider"
+            android:grantUriPermissions="true"
+            android:authorities="${applicationId}.fileprovider"
```

Comment:
Can we call this `${applicationId}.messagingfileprovider` just so we don't clash with any that might exist in the customer's app?